replicate/files.py

```diff
@@ -2,10 +2,136 @@
 import io
 import mimetypes
 import os
-from typing import Optional
+from typing import Any, Dict, List, Optional
 
 import httpx
 
+from replicate.resource import Namespace, Resource
+
+
+class File(Resource):
+    """A file on the server."""
+
+    id: str
+    """The ID of the file."""
```

Comment:
What is the format of the ID? A UUID? Is it worth expounding on that here?

Reply:
I dunno, the ID is opaque, so not really worth mentioning.